gitkraken-client/experimental-features.md

```diff
@@ -0,0 +1,79 @@
+---
+
+title: Experimental Features
+description: Learn about experimental features that are being worked on for possible future inclusion into GitKraken Client.
+taxonomy:
+    category: gitkraken-client
+
+---
+
+Coming from the icy depths of the git ocean are GitKraken Client experimental features. These are ideas we are trying out that still have alot of work ahead, but that we want to share with the world sooner. 
+
+---
+
+## Experimental Features
+
+Navigate to <i class="fas fa-cog"></i><kbd><strong>Preferences</strong><i class='fa fa-caret-right'></i><strong>Experimental</strong></i></kbd> to access the experimental features menu.
+
+Experimental Features are still under development - treat these as an early sneak peek at some of the new functionality we're working on at GitKraken. Experimental features may not work as intended and could be changed or removed in the future. These settings are entirely optional and can be turned off at any time.
+
+If you do experience issues, or have any other feedback, please reach out to us [Contact Support](https://help.gitkraken.com/gitkraken-client/contact-support/?issue_category__customer_facing_field_=Experimental+feedback&subject=GitKraken+Client+Experimental+feedback).
+
+### Git Executable
+
+When this setting is enabled, GitKraken will utilize the Git executable instead of the NodeGit library for certain Git actions including fetching and committing. This may provide increased performance and compatibility with certain projects and development environments. This is a partial implentation and will only effect some aspects of Git within GitKraken - the amount of git commands using the git executable will increase with each release.
+
+**Use Git Executable:** ☑ turn on this experimental feature! As soon as you check the box it is working.
```

Comment:
Use font awesome icon:
`**Use Git Executable:** <i class="fa-regular fa-square-check"></i> `

Reply:
Typo: `implentation` should be `implementation`

Reply:
TypoL `only effect` should be `only affect`

Reply:
`Git within GitKraken` should be `Git within GitKraken Client`

Reply:
Need to capitalize g in Git  twice for: `amount of git commands using the git executable will increase with each release.`

Reply:
Change `As soon as you check the box it is working.` to `Check the box to immediately apply the setting.`